### PR TITLE
[Quest API] Add CanRaceEquipItem(item_id) to Perl/Lua.

### PR DESCRIPTION
**Pull request by KinglyKrab**
- Add $mob->CanRaceEquipItem(item_id) to Perl.
- Add mob:CanClassEquipItem(item_id) to Lua.
- Add mob:CanRaceEquipItem(item_id) to Lua.

### CODE REVIEW

**Review by splose — APPROVED** [review]

File: zone/mob.cpp [hunk 1]
```diff
@@ -5653,22 +5653,53 @@ bool Mob::CanClassEquipItem(uint32 item_id)
...
+	const EQ::ItemData* itm = nullptr;
+	itm = database.GetItem(item_id);
 
-	if (GetClass() > 16)
+	if (!itm) {
 		return false;
+	}
+
+	auto item_races = itm->Races;
+	if(item_races == 65535) {
```
Comment (Akkadius) [inline zone/mob.cpp:5688]: This can be `PLAYER_RACE_ALL_MASK` (65535)

File: zone/mob.cpp [hunk 2]
```diff
@@ -5653,22 +5653,53 @@ bool Mob::CanClassEquipItem(uint32 item_id)
...
 		return false;
+	}
+
+	auto item_classes = itm->Classes;
+	if(item_classes == 65535) {
+		return true;
+	}
+
+	auto class_id = GetClass();
+	if (class_id > 16) {
```
Comment (Akkadius) [inline zone/mob.cpp:5666]: This can be `BERSERKER` (16)

**Comment by Akkadius** [comment]
This one has conflicts as well :/

**Comment by KinglyKrab** [comment]
Added constants and fixed conflicts.

**Review by Akkadius — APPROVED** [review]
Fixed conflicts again and waiting on build

**Pull request merged**